src/BugsnagUnity/Payload/Report.cs

```diff
@@ -84,6 +84,12 @@ public void Ignore()
         /// <value>The app.</value>
         public App App => Event.App;
 
+        /// <summary>
+        /// Gets the session data that was gathered for this report.
+        /// </summary>
+        /// <value>The session.</value>
+        public Session Session => Event.Session;
```

Comment:
The spec seems to state that this shouldn't be publicly accessible, and Android also doesn't expose it

Reply:
Fixed 👍